jscomp/bin/bsb.ml

```diff
@@ -9698,14 +9698,14 @@ let output_ninja
     let () =
       output_string oc ninja_required_version ;
       output_string oc "bs_package_flags = ";
-      output_string oc ("-bs-package-name "  ^ package_name);
+      output_string oc ("-bs-package-name "  ^ (Filename.quote package_name));
```

Comment:
Yes it is required, I tried `npm run build` when the `build.ninja` file did not quote the package name and it fails.